server/graphql/resolvers.ts

```diff
@@ -31,12 +31,12 @@ const resolvers = {
     softDeleteClient: (_, { id }, { dataSources}): Promise<ServerResponseInterface> => dataSources.clients.softDelete(id)
   },
   RequestGroup: {
-    numOpen: (parent, __, { dataSources }): Number => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)).requests.open.length).reduce((total, num) => total + num, 0),
+    numOpen: (parent, __, { dataSources }): Number => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)).requests.length).reduce((total, num) => total + num, 0),
     requestTypes: (parent, __, { dataSources }): Array<RequestTypeInterface> => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)))
   },
   RequestType: {
     requestGroup: (parent, __, { dataSources }): RequestGroupInterface => dataSources.requestGroups.getById(Types.ObjectId(parent.requestGroup.toString())),
-    numOpen: (parent, __, { dataSources }): Number => parent.requests.open.length,
+    numOpen: (parent, __, { dataSources }): Number => parent.requests.length,
```

Comment:
Doesn't this only give the total number of requests? btw i have a change for the numOpen for requestType in my PR LOL

Reply:
wait yeah i think ur PR does cover that.
Do you mind if i merge this in before yours? I just need donor homepage to be working to test deployments from main branch

Reply:
Donor homepage on main is currently broken w/o these changes

Reply:
ya sure. less work for me then :smiley:

Reply:
amazing thnxx kevin :)))

Reply:
parent.requests.filter(request => !request.deleted  && !request.fulfilled).length

I think this would be good